domains/certificates/Query.API/API/DataSyncSyncer/DataSyncSyncerWorker.cs

```diff
@@ -39,6 +39,11 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             var allContracts = await GetAllContracts(stoppingToken);
             foreach (var contract in allContracts)
             {
+                if (contract.EndDate != null && contract.EndDate < DateTime.Now)
```

Comment:
We want to have all measurements within the contract period. Measurements are delayed by several days. This check will mean that we will not get measurements that are delayed.